duckduckgo/iOS
# migrate to Xcode 10.2 and Swift 5

Task/Issue URL: https://app.asana.com/0/392891325557410/1116046264056095
Tech Design URL:
CC:

**Description**:

Updates dependencies to those that compile for Swift 5 and then perform all recommended migrations and fix deprecation warnings.

**Steps to test this PR**:

1. Ensure you're running Xcode 10.2 
1. Clear your Carthage caches - delete the Carthage folder in the project and follow instructions here https://stackoverflow.com/a/45504898/73479 
1. Update to latest Carthage (optional but recommended)
1. Check out this branch and execute `run_tests.sh`
1. Ensure you can run the app - do some exploratory testing 

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
